pkg/server/commands/connected_objects.go

```diff
@@ -163,7 +166,8 @@ func (c *ConnectedObjectsCommand) streamedConnectedObjects(
 	subg, subgctx := errgroup.WithContext(ctx)
 	subg.SetLimit(maximumConcurrentChecks)
 
-	for _, ingress := range ingresses {
+	for i, ingress := range ingresses {
+		span.SetAttributes(attribute.String(fmt.Sprintf("_ingress %d", i), fmt.Sprintf("ingress %s, type %s, tupleset %s", ingress.Ingress.String(), ingress.Type.String(), ingress.TuplesetRelation.String())))
```

Comment:
Let's add a `String()` method on the `RelationshipIngress` type itself and then just do

```go
span.SetAttributes(attribute.String(fmt.Sprintf("_ingress %d", i), ingress.String()))
```